suitcase-{{ cookiecutter.subproject_name }}/suitcase/{{ cookiecutter.subpackage_name }}/__init__.py

```diff
@@ -4,22 +4,10 @@
 
 
 # Suitcase subpackages must follow strict naming and interface conventions. The
-# public API should include some subset of the following. Any functions not
-# implemented should be omitted, rather than included and made to raise
-# NotImplementError, so that a client importing this library can immediately
-# know which portions of the suitcase API it supports without calling any
-# functions.
+# public API should include the following. Any functions not implemented should
+# be omitted, rather than included and made to raise NotImplementError, so that
```

Comment:
While we're editing this, can you fix `NotImplementError` -> `NotImplementedError`?